packages/bits/src/lib/table/table-cell/table-header-cell.component.ts

```diff
@@ -217,7 +221,15 @@ export class TableHeaderCellComponent extends CdkHeaderCell implements OnInit, O
         this.subscriptions.push(this.tableStateHandlerService.columnWidthSubject.subscribe(() => {
             const columnWidth = this.tableStateHandlerService.getColumnWidth(this.columnDef.name);
             if (columnWidth > 45) {
-                this.elementRef.nativeElement.style.width = columnWidth + "px";
+                if (this.tableStateHandlerService.getColumnWidthFixed(this.columnDef.name)) {
```

Comment:
It looks like we're missing the original code from before:

```suggestion
                elem.style.width = columnWidth + "px";
                
                if (this.tableStateHandlerService.getColumnWidthFixed(this.columnDef.name)) {
```